File: LeetCode/array/p1/C++/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> i_map;
        for(int i=0;i<nums.size();i++){
            int i_second = target-nums[i];
            if(i_map.find(i_second)!=i_map.end()){
                return {i_map[i_second],i};
            }
            i_map.insert(pair<int, int>(nums[i], i));
        }
        return {0,0};
    }
};

File: LeetCode/array/p1/java/Solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] i_result = new int[2];
        for(int i=0;i<nums.length-1;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    i_result[0]=i;
                    i_result[1]=j;
                    return i_result;
                }
            }
        }
        return i_result;
    }
}